### ISSUE TITLE
Ansible based Uberspace provisioning

### ISSUE THREAD

**Issue by tpraxl** [Uberspace/lab#1437]
Some time ago, I created an [ISC](https://choosealicense.com/licenses/isc/) licensed collection of [Ansible](https://www.ansible.com/) roles for repeated Uberspace provisionings.

I use them to setup mail domains, accounts, auto responders, web domains, the php version and more with Ansible.
Some of the roles are biased towards special needs, but all are configurable:

* Collection of modular Ansible roles: https://gitlab.com/macrominds/provision/uberspace
* Group Readme with linked real life examples: https://gitlab.com/macrominds/provision/uberspace/readme

I was wondering if there is any interest in this kind of tooling.
If so, I would be glad to share the projects (although I am not sure if the collection even matches the goals of https://lab.uberspace.de).

With this issue, I would like to gather insight on the potential of interest.

Long story short: I would appreciate your feedback.

**Comment by tpraxl** [comment]
Closing this issue due to a lack of interest.
